### PR TITLE
Private repos

### PR DESCRIPTION
**Pull request by aivogft**
test complete

### CODE REVIEW

**Review by Salles-FA — CHANGES_REQUESTED** [review]
Please fix the Codacy issue

**Comment by Salles-FA** [comment]
https://github.com/tranquilitybase-io/TranquilityBase/issues/66

File: gcpdac_tests/gcp_tests/test_onboarding.py [hunk 1]
```diff
@@ -18,6 +18,12 @@ def test_get_destination_project(self):
...
-            json.loads('{"activatorName": "tb-gcp-hpc-activator", "repoURL":"someurl", "tagName": "sometag"}'))
+            json.loads(
+                '{"repo": {"name": "activator name", '
+                '"url":"https://github.com/tranquilitybase-io/gcp-citrix-activator.git"}, '
+                '"cred": {"user":"git user", "token": "some token"}}'))
         when(json_data).self.assertNotEqual(json_data, {"key1": "value1"}).thenReturn(
             self.fail("Failed to return valid return url"))
+
+
+    #git clone --progress -v https://aivogft:a2a0076e37f41f706c052ccef1e15a81c5987f19@github.com/tranquilitybase-io/gcp-citrix-activator.git
```
Comment (Salles-FA) [inline gcpdac_tests/gcp_tests/test_onboarding.py:29]: @aivogft, I'm sure you have already revoked this token. Please remove from the code. Thx!

**Review by Salles-FA — CHANGES_REQUESTED** [review]

**Comment by Salles-FA** [comment]
image

**Review by sunderhill-gft — APPROVED** [review]
Looks good

**Review by Salles-FA — APPROVED** [review]

**Pull request merged**